### ISSUE TITLE
Reproduction of the numbers in the paper

### ISSUE THREAD

**Issue by hongsukchoi** [mkocabas/VIBE#151]
Thanks again for your great work.
But I found **two critical bugs**.

1. While the paper says VIBE uses a bi-directional GRU, the code and model weights you provide are set to uni-directional GRU.
I think this is a critical difference, and either the code and model or the paper should be revised.
Besides, I can't reproduce the numbers on 3DPW as https://github.com/mkocabas/VIBE/issues/99#issue-650045015.
When I train VIBE using bi-GRU with the code and datasets provided here, I get 
`MPJPE: 92.3440, PA-MPJPE: 57.3929, PVE: 108.6150, ACCEL: 25.3047, ACCEL_ERR: 26.3427`, 
which is worse than the baseline.

2. I can't get the same numbers on MPI-INF-3DHP test set with your pre-trained model weights (`vibe_model_wo_3dpw.pth.tar`).
I evaluated 17 joints of valid frames of MPI-INF-3DHP  test set aligning the hip joint, and got
`MPJPE: 101.1546, PA-MPJPE: 65.2166`.
This is clearly worse than `MPJPE: 97.7 PA-MPJPE: 63.4` which is reported in your paper.
Also, VIBE's PCK (`89.0`) is too high compared to SPIN's (`76.4`).
Could you explain VIBE's the evaluation process on MPI-INF-3DHP  test?

Thank you.

**Comment by uyoung-jeong** [comment]
I got different result. MPJPE: 99.1523, PA-MPJPE: 64.8514, PVE: 912.4414, ACCEL: 31.4988, ACCEL_ERR: 31.2689. Still not the same as the paper.
